### PR TITLE
add 'polished' package

### PR DESCRIPTION
**Pull request by phoward38**
For adding tutorials, packages, or apps
===========================================

##### What did you add to the README.md file?
- package - add `polished` package
- contributor - add name & URL

Have you ensured your contribution adheres to the pull request guidelines of [awesome](https://github.com/sindresorhus/awesome/edit/master/pull_request_template.md)? In particular:

- Not a duplicate.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Drop all the `A` / `An` prefixes in the descriptions.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.

----------

##### Do you want to be added to the contributors list on the README.md file?  

- [x] Yes
- [ ] No

##### What name would you like?
Patrick Howard

##### What website would you like your name linked to?
[https://polished.tech](https://polished.tech)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -165,6 +165,7 @@ A curated list of resources for R Shiny.  (*Featured on [Awesome-R](https://awes
 - [reactlog](https://github.com/rstudio/reactlog) - Visual insight into a Shiny apps reactive graph.
 - [supreme](https://github.com/strboul/supreme) - Structure Shiny applications developed with modules.
 - [shinymanager](https://github.com/datastorm-open/shinymanager/) - Simple and secure authentication for single 'Shiny' applications using a SQLite database.
+- [polished](https://github.com/Tychobra/polished) - Authentication, user administration, and web hosting for secure Shiny apps & Rmarkdown documents (flexdashboards, HTML, & PDF)
```
Comment (sdhutchins) [inline README.md]:
```suggestion
- [polished](https://github.com/Tychobra/polished) - Authentication, user administration, and web hosting for secure Shiny apps & Rmarkdown documents (flexdashboards, HTML, & PDF).
```

Comment (sdhutchins) [reply README.md]: Just adding a period here @phoward38 ! It looks great.

**Review by sdhutchins — DISMISSED** [review]
Need that period. Then we're good to go.

Thanks for the submission.

File: README.md (hunk 1 above)
Comment (phoward38) [reply README.md]: @sdhutchins Awesome, thanks!

**Review by grabear — APPROVED** [review]
Sorry for the late reply @phoward38 @sdhutchins .  I have been busy at work lately.

**Pull request merged**